OSPSuite.Presentation.Importer/Core/DataSourceFileReaders/ExcelDataSourceFile.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using OSPSuite.Core.Services;
+using OSPSuite.Infrastructure.Import.Services;
+using NPOI.SS.UserModel;
+using System.IO;
+using NPOI.HSSF.UserModel;
+using OSPSuite.Presentation.Importer.Infrastructure;
+
+namespace OSPSuite.Presentation.Importer.Core.DataSourceFileReaders
```

Comment:
Why are all those classes in the Presentation namespace? Aren't they the business logic and not related with presentation at all?

Reply:
We started by putting everything together. Truth be told we did not understand the separation on presentation/UI that existed before so we were planning to discuss with you guys and then move the classes to their final projects